psi-ms.obo

```diff
@@ -23617,6 +23617,208 @@ relationship: has_metric_category MS:4000008 ! ID based metric
 relationship: has_metric_category MS:4000016 ! retention time metric
 relationship: has_value_type xsd:float ! The allowed value-type for this CV term
 
+[Term]
+id: MS:4000151
+name: MsQuality
+def: "MsQuality – an interoperable open-source package for the calculation of standardized quality metrics of mass spectrometry data." [DOI:10.1101/2023.05.12.540477, https://github.com/tnaake/MsQuality/]
```

Comment:
just a very loose suggestion:
use an organization (e.g. https://github.com/MS-Quality-Hub) to host the package, rather than your personal account. This keeps the URL valid, no matter who maintains it in the future.